liferay-echo/liferay-portal
# [TS] Friendly url input is not loaded correctly when web content's language is different from site's language

Motivation
=======
Friendly url input's language is not loaded correctly when web content's language is different from site's language. Please check:https://liferay.atlassian.net/browse/LPS-203255

Proposed Solution
========
- Add a new optional field "defaultLanguageId" that can be set (similar to other localized inputs) to friendly-url input.
   - If this field is not present, old behavior is used (site's default language).
- Update Journal administration to use this new field.

Steps to Verify
========
Steps to reproduce:
1. Run Liferay and enable system settings “Web Content > Administration >Changeable Default Language“.
2. Check en_US is default site language.
3. Create a Basic Web content and modify default language to es_ES with some content, press save.
4. Reopen the web content for editing.

**Expected**: 
- All the fields are prepopulated with the default web content language, including friendly url.

**Current**: 
- The friendly url shows the site’s default language (en_US) and not the web content’s language (es_ES). 
- Switching the content’s languages makes the friendly url editable, but the default friendly-url language should be “es_ES” and is marked as non-default.

Alternative Solutions that were discarded
========
- Include "selectedLanguageId" but should not be necessary at this point.
